moveit_core/collision_detector_bullet_description.xml

```diff
@@ -1,4 +1,4 @@
-<library path="collision_detector_bullet_plugin">
```

Comment:
@henningkayser do you know if we want this change?

Reply:
I think "lib" should be omitted. Was there another change in this file so that this could be a conflict leftover?

Reply:
I just checked the diff. In MoveIt 1, the plugin was renamed from `collision_detector_bt_plugin` to `collision_detector_bullet_plugin`. Apparently, this was already fixed in MoveIt 2, so we can drop this change. The "lib/" prefix should be removed

Reply:
I just fixed this, lgtm